cpp/src/parquet/column_writer.h

```diff
@@ -104,12 +104,15 @@ class PARQUET_EXPORT PageWriter {
   // Return the number of uncompressed bytes written (including header size)
   virtual int64_t WriteDictionaryPage(const DictionaryPage& page) = 0;
 
+  /// \brief The total number of bytes written as serialized data and
```

Comment:
What does `serialized data` mean here? Flushed data pages?

Reply:
It means written to sink. In fact, the comment is pasted from below:

```c++
  /// \brief The total number of bytes written as serialized data and
  /// dictionary pages to the ColumnChunk so far
  virtual int64_t total_bytes_written() const = 0;
```

in `ColumnWriter`

Reply:
(By the way, the way `Serialize` may means write to sink, which is like `SerializedPageWriter`. If I'm wrong, please correct me)

Reply:
Simply put
- written: compressed and flushed to sink
- compressed: compressed only and not flushed to sink

Am I correct?

Reply:
Yes